core/arch/arm/include/mm/core_mmu.h

```diff
@@ -703,6 +703,7 @@ struct mmu_partition *core_alloc_mmu_prtn(void *tables);
 void core_free_mmu_prtn(struct mmu_partition *prtn);
 void core_mmu_set_prtn(struct mmu_partition *prtn);
 void core_mmu_set_default_prtn(void);
+void core_mmu_set_default_prtn_tbl(void);
```

Comment:
maybe stubbed when `!defined(CFG_VIRTUALIZATION)`? to remove pragmas in **entry_aXX.S**?

Reply:
This is already stubbed under CFG_VIRTUALIZATION at line number 700.